.github/workflows/lint.yml

```diff
@@ -1,4 +1,4 @@
-name: reviewdog
+name: Lint
```

Comment:
We need to update GitHub Actions badge links in the README.

Reply:
Done in https://github.com/reviewdog/action-tfsec/pull/15/commits/c06c7c5dfcb8ed40c074fe0bc81e36bee82cee8f